ui/lobby/src/view/table.ts

```diff
@@ -1,21 +1,15 @@
-import { h, thunk } from 'snabbdom';
-import { bind, onInsert } from 'common/snabbdom';
+import { h } from 'snabbdom';
+import { bind } from 'common/snabbdom';
 import LobbyController from '../ctrl';
 import { GameType } from '../interfaces';
 import renderSetupModal from './setup/modal';
-import { numberFormat } from 'common/number';
 
 export default function table(ctrl: LobbyController) {
-  const { data, trans, opts } = ctrl;
+  const { trans, opts } = ctrl;
   const hasOngoingRealTimeGame = ctrl.hasOngoingRealTimeGame();
   const hookDisabled =
     opts.playban || opts.hasUnreadLichessMessage || ctrl.me?.isBot || hasOngoingRealTimeGame;
-  const { members, rounds } = data.counters;
   return h('div.lobby__table', [
-    h('div.bg-switch', { attrs: { title: 'Dark mode' } }, [
-      h('div.bg-switch__track'),
-      h('div.bg-switch__thumb'),
-    ]),
```

Comment:
so bg-switch is gone? I thought it was useful.

If we're removing it then there's cleanup to do in other JS/CSS files

Reply:
1vader tried to explain its purpose to me. it only shows for anon users as a shortcut to actually picking dark from dasher when their os transitions to light mode and takes lichess background with it against their wishes. so pretty niche.

It's a beautiful svg / js control. if we ever need it, maybe the top bar would be the place for it.